src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/oops/InstanceKlass.java

```diff
@@ -124,18 +111,6 @@ private static synchronized void initialize(TypeDataBase db) throws WrongTypeExc
     CLASS_STATE_BEING_INITIALIZED = db.lookupIntConstant("InstanceKlass::being_initialized").intValue();
     CLASS_STATE_FULLY_INITIALIZED = db.lookupIntConstant("InstanceKlass::fully_initialized").intValue();
     CLASS_STATE_INITIALIZATION_ERROR = db.lookupIntConstant("InstanceKlass::initialization_error").intValue();
-
-    MISC_REWRITTEN                    = db.lookupIntConstant("InstanceKlass::_misc_rewritten").intValue();
-    MISC_HAS_NONSTATIC_FIELDS         = db.lookupIntConstant("InstanceKlass::_misc_has_nonstatic_fields").intValue();
```

Comment:
These all seem to have been deleted rather than modified ??

Reply:
Yes, they are unused by the SA and not really needed for core dump analysis unless someone finds a reason to see these.  If so, some support can be added at that time.  A better approach would be to have some python code to decode this in gdb if needed (probably only for scratch_class).